src/GraphViewer.js

```diff
@@ -56,8 +56,8 @@ export default class GraphViewer extends React.Component {
                             gType={this.props.gType}
                             gLine1Label={this.props.gLine1Label}
                             gLine2Label={this.props.gLine2Label}
-                            gXAxisLabel={this.props.gXAxisLabel}
-                            gYAxisLabel={this.props.gYAxisLabel}
+                            gXAxisLabel={'L' || this.props.gCobbDouglasKName}
```

Comment:
@nikolas just double checking, this axis changing from 'x' to 'L', but the other one is changing 'y' to 'Y', is that correct?

Reply:
whoops - missed @nbuonin catch here & merged.